src/include/com/amazonaws/kinesis/video/webrtcclient/Stats.h

```diff
@@ -0,0 +1,321 @@
+    ICE_SERVER, //!< Information about the connection to an ICE server (e.g. STUN or TURN)
+    CSRC, //!< Stats for a contributing source (CSRC) that contributed to an inbound RTP stream.
+    DATA_CHANNEL, //!< Stats related to each RTCDataChannel id
+    INBOUND_RTP, //!< Statistics for an inbound RTP stream that is currently received with this RTCPeerConnection object
+    LOCAL_CANDIDATE, //!< ICE local candidate statistics related to the RTCIceTransport objects
+    OUTBOUND_RTP, //!< Statistics for an outbound RTP stream that is currently received with this RTCPeerConnection object
+    PEER_CONNECTION, //!< Statistics related to the RTCPeerConnection object.
+    RECEIVER, //!< Statistics related to a specific receiver and the corresponding media-level metrics
+    REMOTE_CANDIDATE, //!< ICE remote candidate statistics related to the RTCIceTransport objects
+    REMOTE_INBOUND_RTP, //!< Statistics for the remote endpoint's inbound RTP stream corresponding to an outbound stream that
+                        //!< is currently sent with this RTCPeerConnection object
+    REMOTE_OUTBOUND_RTP, //!< Statistics for the remote endpoint's outbound RTP stream corresponding to an inbound stream that
+                         //!< is currently sent with this RTCPeerConnection object
+    SENDER, //!< Statistics related to a specific RTCRtpSender and the corresponding media-level metrics
+    TRACK, //!< Statistics related to a specific MediaStreamTrack's attachment to an RTCRtpSender
+    TRANSPORT, //!< Transport statistics related to the RTCPeerConnection object.
+    SCTP_TRANSPORT, //!< SCTP transport statistics related to an RTCSctpTransport object
+    TRANSCEIVER, //!< Statistics related to a specific RTCRtpTransceiver
+    RTC_ALL //!< Report all supported stats
+} RTC_STATS_TYPE;
+
+/**
+ * @brief Indicates computing states in the checklist
+ * Reference: https://www.w3.org/TR/webrtc-stats/#rtcstatsicecandidatepairstate-enum
+ */
+typedef enum {
+    FROZEN,    //!< A check for the pair hasn't been performed because it is waiting for other checks to complete
+    WAITING,   //!< A check has not been performed for this pair
+    IN_PROGRESS,//!< A check has been sent for this pair
+    FAILED,    //!< A check for the pair was already done and failed
+    SUCCEEDED  //!< A check for the pair was already done and produced a successful result
+} RTC_STATS_ICE_CANDIDATE_PAIR_STATE;
+
+/**
+ * @brief Set details of the IceAgent based on STUN_ATTRIBUTE_TYPE_USE_CANDIDATE flag
+ * Reference: https://www.w3.org/TR/webrtc/#rtcicerole
+ */
+typedef enum {
+    UNKNOWN,    //!< An agent whose role is undetermined. Initial state
+    CONTROLLING,//!< A controlling agent. The ICE agent that is responsible for selecting the final choice of candidate pairs
+    CONTROLLED  //!< A controlled agent. The iCE agent waits for the controlling agent to select the final choice of candidate pairs
+} RTCIceRole;
```

Comment:
All enums are capitalized and underscored. RTC_ICE_ROLE.

Reply:
Done